src/Raven.Server/Documents/Queries/QueryMetadata.cs

```diff
@@ -2317,14 +2319,24 @@ public void HandleSpatial(string methodName, List<QueryExpression> arguments, bo
                             if (spatialExpression.Arguments.Count != 2)
                                 throw new InvalidQueryException($"Method {methodName}() expects first argument to be a point() method with 2 arguments", QueryText, parameters);
 
-                            var latitude = _metadata.ExtractFieldNameFromArgument(spatialExpression.Arguments[0], withoutAlias, "point", parameters, QueryText).Value;
-                            var longitude = _metadata.ExtractFieldNameFromArgument(spatialExpression.Arguments[1], withoutAlias, "point", parameters, QueryText).Value;
+                            var latitudePropertyPath = _metadata.ExtractFieldNameFromArgument(spatialExpression.Arguments[0], withoutAlias, "point", parameters, QueryText).Value;
+                            var longitudePropertyPath = _metadata.ExtractFieldNameFromArgument(spatialExpression.Arguments[1], withoutAlias, "point", parameters, QueryText).Value;
 
                             fieldOptions = new AutoSpatialOptions(AutoSpatialOptions.AutoSpatialMethodType.Point, new List<string>
                             {
-                                latitude,
-                                longitude
+                                latitudePropertyPath,
+                                longitudePropertyPath
                             });
+
+                            if (_metadata.SelectFields != null)
+                            {
+                                latitudePropertyPath = GetPathWithAliasName(latitudePropertyPath);
+                                longitudePropertyPath = GetPathWithAliasName(longitudePropertyPath);
```

Comment:
Aren't you changing them here _after_ the were already included in the `fieldOptions` ?

Reply:
pls see latest commit - code was refactored
either way - I'm not changing the `latitudePropertyPath` & `longitudePropertyPath `
I'm just getting the relevant path for the _metadata which in turn is used by the studio to locate the geodata in the resulting document.